oak_channel/src/frame.rs

```diff
@@ -52,7 +52,7 @@ pub const MAX_BODY_SIZE: usize = MAX_SIZE - BODY_OFFSET;
 #[derive(Clone, Default, Debug)]
 pub struct Frame {
     pub flags: Flags,
-    pub body: Vec<u8>,
+    pub body: Box<[u8]>,
```

Comment:
I am not convinced about this -- what is the goal? Vec<u8> is more common and it does not seem wrong here; are you specifically worried about someone extending a Vec somewhere?